tests/e2e/e2e_test.go

```diff
@@ -86,7 +91,40 @@ func (s *IntegrationTestSuite) TestSlashing() {
 	s.testSlashing(chainAPI)
 }
 
-// todo add fee test with wrong denom order
+func (s *IntegrationTestSuite) TestFeeWithWrongDenomOrder() {
```

Comment:
Looks good. Can you move this to be a function in `e2e_bank_test.go` and invoke it from within the `TestBank` function?